qemu/tests/vsock_test.py

```diff
@@ -0,0 +1,148 @@
+import time
+import logging
+import aexpect
+
+from avocado.utils import process
+
+from virttest import error_context
+from virttest import utils_misc
+
+
+def compile_nc_vsock(session, url):
+    """
+    Download and compile nc_vsock on both host and guest
+
+    :param vm: Object qemu_vm.VM
+    :param url: the url to download nc-vsock
+    :return: Patch to binary nc-vsock or None if compile failed
+    """
+    nc_vsock_dir = '/home/nc_vsock'
+    session.cmd('rm -rf %s' % nc_vsock_dir)
+    process.system('rm -rf %s' % nc_vsock_dir, ignore_status=True)
+    cmd_download = "git clone %s %s" % (url, nc_vsock_dir)
```

Comment:
For minimal install system, no `git` command, and also `wget`.
There have one module `avocado.utils.git`, if you `git.get_repo`, it will try to check `git` command with `path.find_command('git')`.

Reply:
If i understand correctly, only file `nc-vsock.c` is needed for the testing, and it is stable (last update was two years ago.).  So we can move it to deps.

Reply:
@yanan-fu Yes, good idea, thanks.